### PR TITLE
Update paper

### PR DESCRIPTION
**Pull request by JayjeetAtGithub**

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -82,254 +82,113 @@ testing and scientific study validations.
...
+
+### **Resource managers**
+
+Here we talk about the functions of the resource manager
+
+### **Container runtimes**
+
+Here we talk about the container runtimes.
+
+### **Continuous evaluation**
```
Comment (ivotron) [inline paper.md:117]: I think we can leave this out

File: paper.md [hunk 2]
```diff
@@ -82,254 +82,113 @@ testing and scientific study validations.
...
-A set of workflow engines that run on Kubernetes, each having their 
-own syntax, allow users to easily submit workflows to a Kubernetes 
-cluster (assuming they have access to one). Examples of these are 
-Pachyderm [@novella_containerbased_2018], Argo[^argo] and 
-Brigade[^brigade]. The main drawback of these when compared to Popper 
-is the requirement of having access to a Kubernetes cluster.
+## Learning Curve
 
-### **Scientific workflow engines**
+## Related Work
```
Comment (ivotron) [inline paper.md:179]: this would be a top-level section

File: paper.md [hunk 3]
```diff
@@ -82,254 +82,113 @@ testing and scientific study validations.
...
+### **Exascale workflow execution in SLURM clusters**
 
-The problem of implementing multi-container workflows as described in 
-@Sec:intro is addressed by existing tools. We briefly survey some of 
-these technologies and compare them with Popper 2.0 by grouping them 
-in categories.
+Planetscale workflow execution is super computing environments
 
-### **Workflow languages**
+# Results {#sec:result}
```
Comment (ivotron) [inline paper.md:163]: we can include results for each scenario

File: paper.md [hunk 4]
```diff
@@ -82,254 +82,113 @@ testing and scientific study validations.
...
 
-Workflow languages such as CommonWL [@amstutz_common_2016] and Yadage 
-[^yadage] allow users to specify workflows using a standard, 
-runtime-agnostic language that can be implemented by an engine. These 
-languages do not assume a containerized environment, although they 
-support container-based workflows. Compared to the GHA language, their 
-syntax is more complex and they give users more control over execution 
-aspects (e.g. specify scheduling policies).
+## Complexity
```
Comment (ivotron) [inline paper.md:165]: can you expand what do you mean by complexity? I like the idea of quantifying complexity, what did you have in mind?

**Comment by JayjeetAtGithub** [comment]
PopperArchitecture

File: paper.md [hunk 5]
```diff
@@ -54,282 +54,140 @@ While containers solve a big part of the "dependency hell" problem
...
 and reuse.
 
 ![An end-to-end example of a workflow. On the left we have the 
-`.workflow` file that defines the workflow. On the right, a pictorial 
+`.yml` file that defines the workflow. On the right, a pictorial 
 representation of it.
-](./figures/casestudy.pdf){#fig:casestudy}
+](./figures/casestudy.png){#fig:casestudy}
+
+Through this paper, we propose a protocol in which a
```
Comment (ivotron) [inline paper.md:74]: the term "protocol" in CS contexts refers to networking protocols. Maybe we could find another term. 

In general, I think the introduction section is missing the three components we discussed earlier: 1) what's the problem, 2) what's the current state of the art and 3) what are our contributions. For 1) before diving into containers, we can first describe the issues of reproducibility that scientists face. For 2) we can briefly discuss in general the categories of tools that exist (what we'll expand on the related work section). For 3) we can actually list our contributions explicitly, like "The contributions of this paper are:" and then show the list.

**Pull request closed without merging**